optuna/trial/_trial.py

```diff
@@ -96,20 +96,18 @@ def suggest_float(self, name, low, high, *, log=False, step=None):
             Suggest a momentum, learning rate and scaling factor of learning rate
             for neural network training.
 
-            .. testsetup::
+
```

Comment:
Might not need this extra blank line.
```suggestion
```

Reply:
Thanks! I applied your suggestion in https://github.com/optuna/optuna/pull/1221/commits/55a446cbb955effa6103e826e98022f6fb335d90.